### DISCUSSION TITLE
Published 1st Kata ... bit of a problem

### DISCUSSION THREAD

**Discussion by requantage** [codewars/codewars.com#3155] [General]
Hi there. I've just posted my 1st Kata, and I didn't realise my validation solution would be published as well. Can I delete it? Delete the whole Kata?

The code is a big, messy work in progress, which isn't great, but there's a bigger problem: the Kata is part 1 of 2, and the code contains a partial solution to part 2, and lots of clues as to how the problem is designed .

What's the best way forward?

**Comment by hobovsky** [comment]
You can unpublish this kata and not publish it again until it's still WIP.

**Reply by requantage** [reply]
Thanks. I've now unpublished.

Do you know if will be able to republish the same Kata with the solution edited (hiding the original), or is that solution now locked in?

**Reply by requantage** [reply]
The solution was so simple ...

Just edit the solution code and republish ... done!
